### PR TITLE
Add MySQL types (C++) from TC

### LINKED ISSUE #89: Missing page for Datatype correlation
This page is very useful https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2130058/MySQL+types+C+

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/75744484-missing-page-for-datatype-correlation?utm_campaign=plugin&utm_content=tracker%2F126289992&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F126289992&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Kitzunu**
closes #89 

Rough import from TC if something needs changing just tell me.

### CODE REVIEW

File: docs/MySQLtypesC++.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            <thead>
+                <tr>
+                    <th colspan="4" class="confluenceTh">Mapping from types used in SQL queries</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <th class="confluenceTh">
+                        <p>MySQL type</p>
+                        < /th> <th class="confluenceTh">Trinity type
```
Comment (BarbzYHOOL) [inline docs/MySQLtypesC++.md]:
```suggestion
                        <th class="confluenceTh">Core type
```

https://github.com/azerothcore/wiki/blob/7f08992aacde1c7a5a7ed810b22219658c0a6b5e/docs/MySQLtypesC%2B%2B.md

**Comment by BarbzYHOOL** [comment]
thank you!!

**Comment by BarbzYHOOL** [comment]
thank you!!

**Comment by BarbzYHOOL** [comment]
why call it "mysql types (c++)"

i don't get how mysql and c++ go along together?

**Comment by FrancescoBorzi** [comment]
because there are examples on how the MySQL types map to C++, e.g.:

```
int32 val = fields[n].GetInt32();
```

**Comment by BarbzYHOOL** [comment]
Ok so let's merge ? @Helias @FrancescoBorzi

**Comment by FrancescoBorzi** [comment]
ok for me

**Comment by BarbzYHOOL** [comment]
ty kitzunu

**Comment by Kitzunu** [comment]
I will look into the issue it made, didnt see it when I tested initially

**Comment by BarbzYHOOL** [comment]
which issue?

**Comment by BarbzYHOOL** [comment]
@Yehonal please, have a look maybe it's a bug in the wiki, and that's why we need a way to preview the changes in the wiki :'(

- When someone makes a page, we check the preview in github markdown https://github.com/Kitzunu/wiki/blob/d06f93231609093c614a3a4000d8da4a61d7238b/docs/MySQLtypesC%2B%2B.md result = fine
- But now look at the page http://www.azerothcore.org/wiki/MySQLtypesC++ fucked up!!!

**Pull request merged**